src/Parser.zig

```diff
@@ -1780,6 +1779,10 @@ fn recordSpec(p: *Parser) Error!Type {
         p.syms.syms.items(.ty)[symbol_index.?] = ty;
     }
 
+    if (!ty.hasIncompleteSize()) {
+        RecordLayout.recordLayout(&ty, p.pp.comp, p);
```

Comment:
```suggestion
        record_layout.compute(&ty, p.pp.comp, p);
```
Style nit + avoiding repetition.